docs/docs/using-gatsby-image.md

```diff
@@ -37,25 +37,56 @@ With Gatsby, we can make images way way better.
 
 `gatsby-image` is designed to work seamlessly with Gatsby’s native image processing capabilities powered by GraphQL and Sharp. To produce perfect images, you only need to:
 
-1. Import gatsby-image and use it in place of the built-in img
-1. Write a GraphQL query using one of the included GraphQL “fragments” which specify the fields needed by `gatsby-image`.
+1. Install `gatsby-image` and its peer dependencies. The `gatsby-config.json` needs to include the following:
+
+```js:title=src/gatsby-config.js
```

Comment:
The config file is in the root, not in src :)

Reply:
Might have messed up the connections between the PRs, but updated the code to address this and below comment: https://github.com/gatsbyjs/gatsby/pull/11998